generators/server/templates/src/main/java/package/security/oauth2/CustomClaimConverter.java.ejs

```diff
@@ -83,6 +83,15 @@ public class CustomClaimConverter implements Converter<Map<String, Object>, Map<
                 if (user.has("family_name")) {
                     convertedClaims.put("family_name", user.get("family_name").asText());
                 }
+                if (user.has("email")) {
+                    convertedClaims.put("email", user.get("email").asText());
+                }
+                // Allow full name in a name claim - happens with Auth0
+                if (user.has("name")) {
+                    String[] name = user.get("name").asText().split("\\s+");
```

Comment:
By default, I guess for new users, `email` value is assigned to the `name` field on the `Auth0` side. If you update the `name` field value, then you can't be sure if that also contains the last name. I suggest including a length check before reading index-based values.